pkg/blackduck/ctl_blackduck.go

```diff
@@ -120,39 +112,10 @@ const (
 	IPV6DisabledSpec                    string = "IPV6Disabled"
 )
 
-// SetPredefinedCRSpec sets the blackDuckSpec to a predefined spec
-func (ctl *CRSpecBuilderFromCobraFlags) SetPredefinedCRSpec(specType string) error {
-	switch specType {
-	case EmptySpec:
-		ctl.blackDuckSpec = &blackduckv1.BlackduckSpec{}
-	case PersistentStorageLatestSpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultPersistentStorageLatest()
-	case PersistentStorageV1Spec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultPersistentStorageV1()
-	case ExternalPersistentStorageLatestSpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultExternalPersistentStorageLatest()
-	case ExternalPersistentStorageV1Spec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultExternalPersistentStorageV1()
-	case BDBASpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultBDBA()
-	case EphemeralSpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultEphemeral()
-	case EphemeralCustomAuthCASpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultEphemeralCustomAuthCA()
-	case ExternalDBSpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultExternalDB()
-	case IPV6DisabledSpec:
-		ctl.blackDuckSpec = util.GetBlackDuckDefaultIPV6Disabled()
-	default:
-		return fmt.Errorf("Black Duck spec type '%s' is not valid", specType)
-	}
-	return nil
-}
-
 // AddCRSpecFlagsToCommand adds flags to a Cobra Command that are need for BlackDuck's Spec.
 // The flags map to fields in the CRSpecBuilderFromCobraFlags struct.
 // master - if false, doesn't add flags that all Users shouldn't use
-func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Command, master bool) {
+func (ctl *HelmValuesFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Command, master bool) {
```

Comment:
Can we remove the DesiredState flag? It has been deprecated for a while now: https://github.com/blackducksoftware/synopsys-operator/pull/1293/files#diff-2ebafce1624553553e121ddbe1c87469R165